extensions/sql-database-projects/src/dialogs/publishDatabaseDialog.ts

```diff
@@ -905,6 +911,42 @@ export class PublishDatabaseDialog {
 		return loadProfileButton;
 	}
 
+	private createSaveProfileAsButton(view: azdataType.ModelView): azdataType.ButtonComponent {
+		let saveProfileAsButton: azdataType.ButtonComponent = view.modelBuilder.button().withProps({
+			label: constants.saveProfileAsButtonText,
+			title: constants.saveProfileAsButtonText,
+			ariaLabel: constants.saveProfileAsButtonText,
+			width: cssStyles.PublishingOptionsButtonWidth,
+			height: '25px',
+			secondary: true
+		}).component();
+
+		saveProfileAsButton.onDidClick(async () => {
+			const filePath = await vscode.window.showSaveDialog(
+				{
+					defaultUri: vscode.Uri.file(path.join(this.project.projectFolderPath, `${this.project.projectFileName}_1`)),
```

Comment:
if a profile has already been loaded, shouldn't the `defaultUri` be the same as the file that was loaded? As a follow up to this PR, it would also be nice if the number can be incremented if a file with that path already exists (and isn't the one that was loaded)